contracts/samples/gnosis/EIP4337Manager.sol

```diff
@@ -75,10 +80,32 @@ contract EIP4337Manager is GnosisSafe, IAccount {
         bytes memory data,
         Enum.Operation operation
     ) external {
-        address msgSender = address(bytes20(msg.data[msg.data.length - 20 :]));
-        require(msgSender == entryPoint, "account: not from entrypoint");
-
-        (bool success, bytes memory returnData) = execTransactionFromModuleReturnData(to, value, data, operation);
+        address _msgSender = address(bytes20(msg.data[msg.data.length - 20 :]));
```

Comment:
for consistency, we removed "_" from members (used only for internal functions, or in case there is a real use-case (e.g. shadowing a parameter)